src/components/contracts/Detail.vue

```diff
@@ -11,7 +11,7 @@
               <div v-else class="sc-no-logo-detail">C</div>
             </div>
             <div class="sc-detail-desc">
-              <h4>{{ contract.list.name }}</h4>
+              <h4>{{ contract.list.name.length !== 0 ? contract.list.name : contract.list.contract_hash }}</h4>
```

Comment:
This change is no longer necessary
https://scaddr.apriorit.com/jira/browse/ONXP-1588?focusedCommentId=27643&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-27643